### PR TITLE
Use symlink for output_template/python/lmnet/utils/output.py

### PR DESCRIPTION
**Pull request by yd8534976**
Related to: #684 .

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by tsawada** [comment]
Please put me back after ownership review.

**Comment by yd8534976** [comment]
@tsawada , Thanks!
Since there are some differences between two files.
1. For example, we have no `matplotlib` in `output_template/requirements.txt`.
https://github.com/blue-oil/blueoil/blob/02598dd1710ceb62a59e0e30da33bee85b0ef505/output_template/python/requirements.txt#L1-L9

2. There is no `bench` in `predict_output/output.py`'s `__init__()`.
https://github.com/blue-oil/blueoil/blob/02598dd1710ceb62a59e0e30da33bee85b0ef505/output_template/python/lmnet/utils/output.py#L36-L42
https://github.com/blue-oil/blueoil/blob/02598dd1710ceb62a59e0e30da33bee85b0ef505/lmnet/lmnet/utils/predict_output/output.py#L38-L43

So I synchronized it manually.

File: lmnet/lmnet/utils/predict_output/output.py [hunk 1]
```diff
@@ -35,12 +34,13 @@ class JsonOutput():
     Please see [Output Data Specification](https://github.com/LeapMind/lmnet/wiki/Output-Data-Specification).
     """
 
-    def __init__(self, task, classes, image_size, data_format):
+    def __init__(self, task, classes, image_size, data_format, bench={}):
```
Comment (tsawada) [inline lmnet/lmnet/utils/predict_output/output.py]: Don't use mutable object as an argument's default value
http://go/python-review-guidelines#using-listdict-as-the-default-arg

File: lmnet/lmnet/utils/predict_output/output.py [hunk 2]
```diff
@@ -16,15 +16,14 @@
...
 import json
 import os
-from datetime import datetime, timezone
+from datetime import datetime
 from io import BytesIO
 
 import numpy as np
 import PIL.Image
 import PIL.ImageDraw
-from matplotlib import cm
```
Comment (tsawada) [inline lmnet/lmnet/utils/predict_output/output.py]: can we remove `matplotlib` from `requirements.txt`?

File: lmnet/lmnet/utils/predict_output/output.py (hunk 1 above)
Comment (yd8534976) [reply lmnet/lmnet/utils/predict_output/output.py]: Thanks, it is reasonable. Fixed.

File: lmnet/lmnet/utils/predict_output/output.py (hunk 2 above)
Comment (yd8534976) [reply lmnet/lmnet/utils/predict_output/output.py]: After this PR, only `lmnet/networks/classification/base.py` will depend on `matplotlib`.
https://github.com/blue-oil/blueoil/blob/8055a4a29618d79cf70d068d5aac5af7a4f3351e/lmnet/lmnet/networks/classification/base.py#L17
Is it worth adding a new dependency for visualization? If not, I would remove it in another PR and reimplement this.
Related to: https://github.com/blue-oil/blueoil/issues/485

Comment (tsawada) [reply lmnet/lmnet/utils/predict_output/output.py]: I see. I think it'd be nice.

**Pull request merged**